clang/lib/CodeGen/Targets/Hexagon.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "ABIInfoImpl.h"
 #include "TargetInfo.h"
+#include "llvm/IR/DerivedTypes.h"
```

Comment:
I don't think the extra include is needed?

Reply:
Likely not, I think my LSP automatically inserts them when it completes stuff like PointerType::get. I'll remove it and the ones in the other PR too